server/channels/app/webhook.go

```diff
@@ -109,11 +108,14 @@ func (a *App) TriggerWebhook(c request.CTX, payload *model.OutgoingWebhookPayloa
 		contentType = "application/x-www-form-urlencoded"
 	}
 
+	var wg sync.WaitGroup
 	for i := range hook.CallbackURLs {
+		wg.Add(1)
 		// Get the callback URL by index to properly capture it for the go func
 		url := hook.CallbackURLs[i]
 
-		a.Srv().Go(func() {
+		go func() {
+			defer wg.Done()
```

Comment:
Can you talk a bit about how these are different? To me they look pretty similar (except that one is using a wg, the other an atomic).

Reply:
In the previous case, it would only wait during server shutdown for all goroutines to finish. With this change, we are waiting for all goroutines to finish before the function exits. Granted, both are effectively the same because `handleWebhookEvents` is itself run in a goroutine. Happy to revert it if you prefer.